setup.py

```diff
@@ -9,8 +9,8 @@
 with open(path.join(here, "requirements.txt")) as f:
     requirements = []
     for line in f:
-        # Skip comment lines
-        if not line.startswith("#"):
+        # Skip comment lines and hash values
+        if (not line.strip().startswith("#") and not line.strip().startswith("--")):
```

Comment:
Now that we're not commenting out hashes, you can get rid of the first part of this if statement. I think you'll need to add the new part to agent/setup.py, too.